db/migrate/20200723225140_create_campaigns.rb

```diff
@@ -0,0 +1,16 @@
+class CreateCampaigns < ActiveRecord::Migration[6.0]
+  def change
+    create_table :campaigns do |t|
+      t.boolean :active, default: false
+      t.boolean :is_valid, default: false
+      t.datetime :end_date, null: false
+      t.string :description, null: false
+      t.string :gallery_image_urls, default: [], null: false, array: true
```

Comment:
This could also be nullable. Maybe we can put a default gallery image in the frontend